src/compiler/contextualize.ml

```diff
@@ -1,39 +1,82 @@
 open Surface.Ast
 
-module Acc = struct
+module Acc : sig
```

Comment:
You can find a better name than `Acc` no?

Reply:
Probably, but nothing that convey properly what it is that won't immediately become an eyesore in the code, and be aliased away.